rules/sort-modules.ts

```diff
@@ -296,11 +297,12 @@ let analyzeModule = ({
           break
         default:
       }
+
+      return !!selector && !!name
     }
     /* eslint-enable @typescript-eslint/no-loop-func */
-    parseNode(node)
 
-    if (!selector || !name) {
+    if (!parseNode(node)) {
```

Comment:
[readability] I like it the other way
- We have to read `parseNode` to see that the function checks that `selector` and `name` are defined to return `true`.
- Typescript doesn't know that, so we need to add `!` assertions to `name` and `selector`.

Reply:
Back to the previous version.